.github/workflows/create-jira-chore-issue.yml

```
@@ -0,0 +1,90 @@
+name: Create Jira Chore Issue
+on:
+  issues:
+    types:
+      - opened
+
+jobs:
+  create-issue:
+    if : "startsWith(github.event.issue.title, 'Chore')"
```

YAMLlint (1.35.1)

[warning] too many spaces before colon

(colons)